arviz/plots/parallelplot.py

```diff
@@ -69,9 +77,17 @@ def plot_parallel(
     var_names, _posterior = xarray_to_ndarray(
         get_coords(posterior_data, coords), var_names=var_names, combined=True
     )
-
     if len(var_names) < 2:
         raise ValueError("This plot needs at least two variables")
+    if normalize:
+        if norm_method == "normal":
+            _posterior = (_posterior - np.mean(_posterior)) / np.std(_posterior)
```

Comment:
Is this OK, or the axis need to be specified?

Reply:
Mean and S.D of an individual row should be used, however I'll make the changes after further discussion.

Reply:
Specified the axis.